inc/saimacsec.h

```diff
@@ -292,6 +292,31 @@ typedef enum _sai_macsec_attr_t
      */
     SAI_MACSEC_ATTR_AVAILABLE_MACSEC_SA,
 
+    /**
+     * @brief Terminate/Forward Pause Frames.
+     *
+     * If enabled (true) pause frames are forwarded to the user application.
+     * In normal mode (false) pause frames are terminated and discarded within the MAC.
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_MACSEC_ATTR_PAUSE_FRAME_FORWARD_ENABLE,
```

Comment:
this is not specific to macsec, can you move these two new attributes to SAI_PORT_ATTR ?